### ISSUE TITLE
Can't have 2 controllers connected at the same time

### ISSUE THREAD

**Issue by 10c8** [psmoveservice/PSMoveService#693]
I've paired both of my **ZCM1U** controllers to my PC and configured them using `PSMoveConfigTool`. Each controller works fine, as long as **_just one_ is turned on and connected**. A few seconds after I press the PS button on the other controller and it connects to PSMoveService, I hear the "device disconnected" sound from Windows, the Bluetooth dongle's light turns off, both controllers disconnect, then I hear the "device connected" sound and the dongle's light turns on again.

_(I know there's currently an issue regarding Bluetooth devices on Windows 10 versions above 1903, but I remember having the same issue a year or two ago, on an earlier version)._

**TLDR:** Connecting more than a single controller on Bluetooth apparently crashes the dongle.

**PSMoveService version:** alpha_0.0.1
**Windows 10 version:** 1909 (paired the controllers using a custom build, as Bluetooth seems broken on this version)

**Comment by ZedVR** [comment]
Hello!
I have similar problem, although I am still using win7 (I know I know, but I don't like change), but my second controller just simply doesn't connect. Once I turn the connected one off, the other one connects in a second. Couldn't figure out the problem yet, maybe I need a newer bluetooth adapter.

**Comment by AgoniXX** [comment]
Just change to a new BT adapter, my old one has the same issue.
I just bought the most popular one on amazon, it works great:
https://www.amazon.com/ZEXMTE-Bluetooth-Receiver-Transfer-Wireless/dp/B0775YF36R/

**Comment by lmnope1** [comment]
With Gen 2 controllers there is a similar issue - when one is connected the other will not connect.
